components/geometry-merger/examples/buffer/index.html

```diff
@@ -4,17 +4,27 @@
     <meta charset="utf-8">
     <title>A-Frame Geometry Merger Component - Buffer</title>
     <meta name="description" content="Merging buffer geometry with Geometry Merger component."></meta>
-    <script src="https://aframe.io/releases/0.8.2/aframe.js"></script>
+    <script src="https://aframe.io/releases/1.4.1/aframe.js"></script>
+    <script>
+      // This is a patch for an A-Frame bug not yet fixed in the latest published version:
```

Comment:
Can be deleted once we have a release that includes #5250 & we update to it. For now this is needed.